tests/integration/Segments/DynamicSegments/Filters/WooCommerceNumberOfOrdersTest.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types = 1);
```

Comment:
I have seen some test files with strict_types enabled and some with strict_types disabled. I'd be curious to learn more about why you added here and if there is a standard that we follow that says that we should enable strict_types in some circumstances.

Reply:
If I remember correctly, we would prefer to use strict types in new classes. So when I edit an older class, I enable it too sometimes.